build/build.sh

```diff
@@ -145,6 +145,12 @@ dascandy-contracts-trunk)
     VERSION=dascandy-contracts-trunk-$(date +%Y%m%d)
     LLVM_ENABLE_RUNTIMES+=";libunwind"
     ;;
+ericwf-contracts-trunk)
+    BRANCH=contracts-nightly
+    URL=https://github.com/efcs/llvm-project
+    VERSION=ericwf-contracts-trunk-$(date +%Y%m%d)
+    LLVM_ENABLE_RUNTIMES+="libcxx;libcxxabi;libunwind"
```

Comment:
missing `;` at the front if you want to send a new PR!

Reply:
Sorry about that. Sent.

Reply:
sorry I didn't spot it!